lib/brcobranca/calculo.rb

```diff
@@ -71,10 +71,52 @@ def modulo11_9to2_10_como_x
     #
     # @return [Integer]
     def modulo11_9to2_10_como_zero
-      valor = self.modulo11_9to2
+      valor = modulo11_9to2
       valor == 10 ? 0 : valor
     end
 
+    def modulo11_mercantil
+      # valor = self.modulo11_2to9
+      num = 2
+      array = []
+      1.upto(self.size) do |i|
```

Comment:
Unused block argument - `i`. You can omit the argument if you don't care about it.<br>Redundant `self` detected.